packages/core/src/baseclient.ts

```diff
@@ -141,6 +150,17 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
     return eventId;
   }
 
+  /**
+   * @inheritDoc
+   */
+  public captureSession(session: Session): void {
+    if (!session.release) {
+      logger.warn('Discarded session because of missing release');
```

Comment:
Could we possibly check for this on `init`? If release is not set, then disable sessions with a warning.

Would reduce the logger noise once we start sending regular session updates.

Reply:
I see. A bit questionable if code reports different releases though, or doesn't know the release at init time... Looks like a sharp-edge API to ask for a release in more than one place in the SDK.